installers/vagrant-dev/radiasoft-download.sh

```diff
@@ -171,18 +186,25 @@ EOF
     fi
     vagrant ssh <<EOF
 $(install_vars_export)
-curl $(install_depot_server)/index.sh | bash -s redhat-dev
+curl $(install_depot_server)/index.sh | \
```

Comment:
Quote `"$os"`. 

Ideally, we should encapsulate the curl, index.sh in a function that is something like:

```sh
export bivio_home_env_ignore_git_dir_ownership=$(vagrant_dev_ignore_git_dir_ownership $os) 
$(install_repo_in_subshell redhat-dev)
```

This would implicitly set install_vars_export, and do the curl appropriately, possibly also setting `-euo`. For another day, but noting that there's a lot of code here, unnecessarily.

I like the export on its own line since it makes the code less messy.

Reply:
Agreed. Just the quote for now b2d113e